src/pygeoprocessing/geoprocessing_core.pyx

```diff
@@ -734,8 +734,9 @@ def _raster_band_percentile_int(
         pixels_processed += block_data.size
         if time.time() - last_update > 5.0:
             LOGGER.debug(
-                'data sort to heap %.2f%% complete',
-                (100.*pixels_processed)/n_pixels)
+                f'data sort to heap {(100.*pixels_processed)/n_pixels}% '
```

Comment:
Hey Rich, can we format this float percentage doing something like `f'{value:.2f}'`. So:
`f'data sort to heap {((100.*pixels_processed)/n_pixels):.2f}% '`

Reply:
fixed in 4dab739